app/styles/_shame.scss

```diff
@@ -207,4 +207,11 @@ cursor {
 .u-overflow-y-scroll {
   overflow-y: scroll;
   height: 100%; 
+}
+
+.u-overflow-divider {
+  * + & {
+    border-top: .1rem solid var(--au-gray-200);
+    margin-top: $au-unit;
```

Comment:
nitpick, but on tablet landscape the scroll area is still fairly small. 
![image](https://github.com/lblod/frontend-burgernabije-besluitendatabank/assets/3533236/878a9c32-eb78-4e22-ad31-07ec9b52d27c)

I think the top part can be condensed a bit more maybe?

Reply:
True, reduced it a bit more. Still not ideal as font-sizes are also a bit big here. Will explore further in a separate PR.

Reply:
Awesome, thanks!